### DISCUSSION TITLE
Placing objects in the floor

### DISCUSSION THREAD

**Discussion by SAYtan47** [FakeFishGames/Barotrauma#14614] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

While placing components like water detectors/ garden pots theres a posibility to place it too deep in the floor and after that it will be uninteractable and practically wasted
![20240906145701_1](https://github.com/user-attachments/assets/5cf4016c-f7b5-4039-8ca4-77c603ed3936)

### Reproduction steps

1. Start a test in submarine editor
2. Give yourself water detector or a garden pot
3. Lead your cursor as low down as possible (might have to sit for it)
4. Under some angles the possibility to place item lower than usual will occur

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by ubertpendragon** [comment]
Can also be a problem in ruin walls with scanners.  

`interactthroughwalls="true"` might help

**Comment by Regalis11** [comment]
This has been fixed in our private repo now, fix coming up in a future update!
